### PR TITLE
Add i18n locale information for Uganda, Kenya and Tanzania.

### PR DESCRIPTION
**Pull request by bahiirwa**
### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:
Add i18n locale information for Uganda, Kenya and Tanzania.

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [x] Have you successfully run tests with your changes locally?

### Changelog entry
Add i18n locale information for Uganda, Kenya and Tanzania.

### CODE REVIEW

**Review by rrennick — DISMISSED** [review]
This looks good other than needing to be rebased.

**Comment by bahiirwa** [comment]
Rebase done @rrennick

File: i18n/locale-info.php [hunk 1]
```diff
@@ -162,13 +162,21 @@
...
 		'dimension_unit' => 'cm',
 	),
+	'KE' => array(
+		'currency_code'  => 'KES',
+		'currency_pos'   => 'left',
+		'thousand_sep'   => ',',
+		'decimal_sep'    => '.',
+		'num_decimals'   => 0,
+    'weight_unit'    => 'kg',
+		'dimension_unit' => 'cm',
```
Comment (rrennick) [inline i18n/locale-info.php:172]: This looks like it's missing the `),` to close the array.

File: i18n/locale-info.php [hunk 2]
```diff
@@ -162,13 +162,21 @@
...
 	'LI' => array(
 		'currency_code'  => 'CHF',
 		'currency_pos'   => 'left_space',
 		'thousand_sep'   => "'",
 		'decimal_sep'    => '.',
 		'num_decimals'   => 2,
-		'weight_unit'    => 'kg',
+    'weight_unit'    => 'kg',
```
Comment (rrennick) [inline i18n/locale-info.php:179]: This looks like a tab has been replaced with spaces.

**Comment by bahiirwa** [comment]
The E2E tests passed the first time and failed when I made a PHPCS change. This PR does not have any code to E2E effect.

**Review by rodrigoprimo — DISMISSED** [review]
Thanks for your contribution.

**Review by rodrigoprimo — APPROVED** [review]

**Pull request merged**